### PR TITLE
DPDK port can now be configured + MTU config

### PR DESCRIPTION
**Pull request by outscale-jju**

### CODE REVIEW

File: api/server/graph.cc [hunk 1]
```diff
@@ -112,10 +112,17 @@ bool Graph::start(std::string dpdk_args) {
     vhost_start();
 
     // Create nic brick
-    nic_ = Brick(pg_nic_new_by_id("port-0", 0, &app::pg_error),
-                 pg_brick_destroy);
+    if (app::config.dpdk_port < 0) {
```
Comment (outscale-mgo) [inline api/server/graph.cc]: you should check `app::config.dpdk_port <= 0` as atoi return 0 when it fail.

File: api/server/app.cc [hunk 2]
```diff
@@ -140,6 +144,8 @@ bool Config::parse_cmd(int argc, char **argv) {
         dpdk_args = std::string(&*dpdk_args_cmd);
     if (nic_mtu_cmd != nullptr)
         nic_mtu = std::string(&*nic_mtu_cmd);
+    if (dpdk_port_cmd != nullptr)
+        nic_mtu = std::atoi(&*dpdk_port_cmd);
```
Comment (outscale-mgo) [inline api/server/app.cc]: here --dpdk-port=two will be valid and will set port to 0.

File: api/server/graph.cc [hunk 3]
```diff
@@ -169,6 +166,57 @@ bool Graph::start(std::string dpdk_args) {
...
 }
 
+void Graph::set_config_mtu() {
+    if (app::config.nic_mtu.length() == 0)
+        goto exit;
+
+    if (app::config.nic_mtu == "max") {
+        app::log.info("try to find maximal MTU");
+        int min = 1400;
+        int max = 65536;
```
Comment (outscale-mgo) [inline api/server/graph.cc]: add a line here pls

File: api/server/app.cc [hunk 4]
```diff
@@ -46,6 +46,7 @@ Config::Config() {
     dpdk_args = DPDK_DEFAULT_ARGS;
     tid = 0;
     nic_mtu = "";
+    dpdk_port = 1;
```
Comment (outscale-mgo) [inline api/server/app.cc]: dpdk_port = 0;

**Comment by outscale-mgo** [comment]
This poll request must be rebase on pg_error if i want it to pass tests, so we need pg_error to be upstream in order to upstream this one.

**Comment by outscale-mgo** [comment]
merge

**Comment by outscale-jju** [comment]
merged

**Pull request closed without merging**